rust/tools/authorization-logic/src/souffle/souffle_interface.rs

```diff
@@ -71,7 +71,7 @@ pub fn ast_to_souffle_file(prog: &AstProgram, output_file_path: &str,
 pub fn run_souffle(input_file_path: &str, queries_outdir: &str) {
     let souffle_command = match std::env::var("SOUFFLE_BIN") {
         Ok(val) => val,
-        Err(e) => "souffle".to_string()
+        Err(_e) => "souffle".to_string()
```

Comment:
Ahh! Shame to not have them. Yes, the _e is to a) silence the warning about an [intentionally] unused binding and b) keep it somewhat clear that the unused value is the error, not something else (`_` would be fine too but I don't think we need to worry about the difference). The somewhat nice thing about `_e` is that it can still be used e.g. `dbg!(_e);` while not being expected to be used, where `_` is pretty likely to be shadowed and so not reliable for this purpose.